public/app/routes/routes.tsx

```diff
@@ -362,6 +362,12 @@ export function getAppRoutes(): RouteDescriptor[] {
         () => import(/* webpackChunkName: "ServerStats" */ 'app/features/admin/ServerStats')
       ),
     },
+    config.featureToggles.onPremToCloudMigrations && {
+      path: '/admin/migrate-to-cloud',
+      component: SafeDynamicImport(
+        () => import(/* webpackChunkName: "MigrateToCloud" */ 'app/features/admin/migrate-to-cloud/MigrateToCloud')
```

Comment:
I'd also add a permission check here. Something like
```
roles: () => ['ServerAdmin'],
```
or
```
roles: () => contextSrv.evaluatePermission([<some_permission>]),
```

There are a bunch of examples of that for other routes if you look around.